roles/import-from-url/tasks/provision.yml

```diff
@@ -35,6 +35,31 @@
 - name: Provision PVC
   command: "oc apply -f /tmp/pvc.yml"
 
+- name: set pvc status jsonpath selectors
+  set_fact:
+    import_status_path: .metadata.annotations.kubevirt\.io/storage\.import\.pod\.phase
+    pvc_status_path: .status.phase
+
+- Name: Check PVC bound status
+  command: "oc get pvc {{ vm_name }}-disk-01 -o jsonpath='{ {{pvc_status_path}} }'"
+  register: pvc_bound_status
+  until: pvc_bound_status.stdout == 'Bound'
+  retries: 10
+  delay: 1
+
+- name: Pod Import Status
+  command: "oc get pvc {{ vm_name }}-disk-01 -o jsonpath='{ {{import_status_path}} }'"
+  register: import_status
+  # The values of import_status follow standard kubernetes pod phase values. Pending/Running/Succeeded/Failed/Unknown
```

Comment:
Here is pod lifecycle [1]. I think that statuses are a bit different.

https://dzone.com/articles/kubernetes-lifecycle-of-a-pod

Reply:
Not according to the kubernetes documentation: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/

Reply:
But for this case the difference is not that important, only interested in Success/Failure of the importer (CrashLoopBackoff/Unknown/Whatever) are not really options. I just looked in the source of the importer pod to see where it go their status information from, and it pointed to pod.phase definition.